OpenRA.Mods.Common/Traits/Voiced.cs

```diff
@@ -32,22 +30,23 @@ public class Voiced : IVoiced
 	{
 		public readonly VoicedInfo Info;
 
+		public string VoiceSet { get; }
```

Comment:
I'm pretty sure `{ get; }` is a getter-only auto-property which was introduced in C# 6 (which OpenRA doesn't support yet).